pkg/diagnostics/service_monitoring.go

```diff
@@ -364,3 +405,68 @@ func (s *serviceMetrics) RequestBlockedByGlobalAction(appID, trustDomain, namesp
 			s.globalPolicyActionBlocked.M(1))
 	}
 }
+
+// ServiceInvocationRequestSent records the number of service invocation requests sent.
+func (s *serviceMetrics) ServiceInvocationRequestSent(appID, destinationAppID, method string) {
```

Comment:
Can we add tests for these methods like the ones recently added in `component_monitoring_test.go`?

Reply:
done